keras/applications/applications_load_weight_test.py

```diff
@@ -69,7 +70,16 @@
         efficientnet_v2.EfficientNetV2B2, efficientnet_v2.EfficientNetV2B3,
         efficientnet_v2.EfficientNetV2S, efficientnet_v2.EfficientNetV2M,
         efficientnet_v2.EfficientNetV2L
-    ])
+    ]),
+    'regnet': (regnet,
```

Comment:
I think you need a tf_py_test entry for this test in the BUILD file, you can see example for other load weights tests.

Reply:
I see.